website/exoscale.erb

```diff
@@ -13,9 +13,12 @@
                 <li<%= sidebar_current("docs-exoscale-data") %>>
                     <a href="#">Data</a>
                     <ul class="nav nav-visible">
+                        <li<%= sidebar_current("docs-exoscale-compute") %>>
+                            <a href="/docs/providers/exoscale/d/compute.html">exoscale_compute</a>
+                        </li>
+
                         <li<%= sidebar_current("docs-exoscale-compute-ipaddress") %>>
                             <a href="/docs/providers/exoscale/d/compute_ipaddress.html">exoscale_compute_ipaddress</a>
-                        </li>
```

Comment:
:+1:

Reply:
Could you be more specific? Thanks.

Reply:
In pure HTML, `<li>` don't have to be closed, which [is **recommended** by Google](https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags)

So lazyness is good as long as the whole document follows the same set of lazy behavior. Otherwise you're hurting consistency.

Here it feels like consistency matters more. The main reviewer will probably agree with me on that.

Reply:
@greut 

Thanks for your comment !
We take note.

This comment would be helpful to many [others](https://github.com/terraform-providers/terraform-provider-google/blob/master/website/google.erb).